vlm/src/main/resources/reference.conf

```diff
@@ -1,4 +1,10 @@
-vlm.geotiff.s3 {
-  allow-global-read: false
-  region: "us-east-1"
+vlm.geotiff ={
+  s3 = {
+    allow-global-read: false
+    region: "us-east-1"
+  }
+
+  hdfs = {
+    resources = ["file:///etc/hadoop/conf/core-site.xml", "file:///etc/hadoop/conf/hdfs-site.xml"]
```

Comment:
mb it should be empty by default? I don't expect everyone willing to set it up explicitly. We can move it into the `test/resource/application.conf` as a usage example.

Reply:
Yeah, this seems reasonable to me.